src/ic-xrc-types/src/lib.rs

```diff
@@ -19,13 +19,24 @@ pub struct Asset {
 }
 
 /// The type the user sends when requesting a rate.
+///
+/// Suppose BTC/USDT is 30_000. This means that you have to give 30_000 USDT in
+/// order to receive 1 BTC. For people with a non-economics background, this is
+/// completely backwards: you would say that the "rate" in this example is
+/// "30_000 USDT per BTC", which would be written as "30_000 USDT/BTC".
+/// Economists do NOT read "BTC/USD" as "BTC per USD". In this example, BTC is
+/// the "base" asset, and USDT is the "quote" asset.
+///
+/// Translation table for non-economists:
+///
+///             X/Y = Ys per X
+///      base asset = resource that you want to receive
+///     quote asset = resource that you are offerring
 #[derive(CandidType, Clone, Debug, Deserialize)]
 pub struct GetExchangeRateRequest {
-    /// The asset to be used as the resulting asset. For example, using
-    /// ICP/USD, ICP would be the base asset.
+    /// The resource being sought.
     pub base_asset: Asset,
-    /// The asset to be used as the starting asset. For example, using
-    /// ICP/USD, USD would be the quote asset.
+    /// The resource being offerred.
```

Comment:
I don't think we should use words like "sought" and "offered".
I think if we define base asset and quote asset above, it should be clear here.

Reply:
Done.

# Tangent.

Today, I am under a new understanding (which might also be incorrect): X/Y does **not** indicate which direction you are going. You could just as well be going from X to Y, or from Y to X. The direction would be indicated elsewhere.

I also learned (thanks to the article that you shared) that X is the "more important" one, but there is no globally agreed upon ranking of the world's currencies.